index.js

```diff
@@ -22,12 +25,15 @@ const interpolate = paraphrase(/\${([^{}]*)}/g, /%{([^{}]*)}/g, /{{([^{}]*)}}/g)
  * @param {Object<Object>}   options.translations JSON compliant object
  * @param {Object<String>}   [options.$scope]     Root string to be use for looking for translation keys
  * @param {Object<Function>} [options.missing]    Method to call when key is not found
+ * @param {Object<Function>} [options.empty]      Method to call when value is empty
```

Comment:
[PETTY]

```suggestion
 * @param {Function} [options.empty]      Method to call when value is empty
```

Same for the other options.